### ISSUE TITLE
Downloaded audiobooks do not work when the internet is disconnected

### ISSUE THREAD

**Issue by ibraelmahallawy** [vipulyaara/Kafka#39]
Downloaded audiobooks do not work when the internet is disconnected

**Comment by vipulyaara** [comment]
Yes, this is a known issue. I did not work on this feature because audio streaming works fine on slower internet, so this was never a strong requirement. I'll prioritize this.

**Comment by vipulyaara** [comment]
This is implemented and should be available in the next release.

**Comment by vipulyaara** [comment]
Releases are halted because the app is taken down from the Play store. But the latest [release](https://github.com/vipulyaara/Kafka/releases/tag/v0.46.0) should have these changes. Feel free to reopen if it doesn't.
